RELEASE-NOTES.md

```diff
@@ -1,5 +1,47 @@
 # Prefect Release Notes
 
+## Release 2.13.6
+
+### New default option for results storage
+
+
+
+See the following pull request for details:
+- https://github.com/PrefectHQ/prefect/pull/10925
+
+### Experimental support for enhanced cancellation
+
+We're introducing a new experimental feature that will enable more consistent and reliable cancellation of flow runs. 
+
+To enable enhanced cancellation, set the 'PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION' setting on your worker or agents to `True`:
+
+```bash
+prefect config set PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION=True
+```
+
```

Comment:
Would it be worth adding a small blurb to the [docs](https://docs.prefect.io/2.13.5/concepts/flows/?h=cancel#cancel-a-flow-run)?

Reply:
Yeah, I'll open a separate PR for that.

Reply:
PR here: https://github.com/PrefectHQ/prefect/pull/10944